interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -14,7 +14,16 @@ angular.module('adage.analyze.sampleBin', [
 
 .factory('SignatureSet', ['$resource', 'ApiBasePath',
   function($resource, ApiBasePath) {
-    return $resource(ApiBasePath + 'node/set/:ids/');
+    return $resource(
+      ApiBasePath + 'node/post_multiple/',
+      {},
+      {post: {
+        method: 'POST',
+        // Setting Content-Type is required. Django will not process the
+        // POST data the way Angular's defaults send it.
+        headers: {'Content-Type': 'application/x-www-form-urlencoded'}
```

Comment:
This was required last time around, but in this case it's not actually encoded as form data or as JSON data. Although it works this way, it's not actually quite right. I'm about to push an update that makes it `'text/plain'` to clarify.